From bcaeba508157200dc580607175e8f2fed65fd181 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 29 Apr 2022 09:19:10 +0100
Subject: [PATCH] staging: vchiq_arm: Add log_level module params

Add module parameters to control the logging levels for the various
vchiq logging categories.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 .../staging/vc04_services/interface/vchiq_arm/vchiq_arm.c    | 5 +++++
 1 file changed, 5 insertions(+)

--- a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
+++ b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
@@ -64,6 +64,11 @@
 /* Run time control of log level, based on KERN_XXX level. */
 int vchiq_arm_log_level = VCHIQ_LOG_DEFAULT;
 int vchiq_susp_log_level = VCHIQ_LOG_ERROR;
+module_param_named(arm_log_level, vchiq_arm_log_level, int, 0644);
+module_param_named(susp_log_level, vchiq_susp_log_level, int, 0644);
+module_param_named(core_log_level, vchiq_core_log_level, int, 0644);
+module_param_named(core_msg_log_level, vchiq_core_msg_log_level, int, 0644);
+module_param_named(sync_log_level, vchiq_sync_log_level, int, 0644);
 
 DEFINE_SPINLOCK(msg_queue_spinlock);
 struct vchiq_state g_state;
